mixins/infra-state.mixin.tf

```diff
@@ -0,0 +1,63 @@
+# <-- BEGIN DOC -->
+#
+# This mixin is meant to be placed in a Terraform config outside the organization's infrastructure monorepo in order to:
+#
+# 1. Instantiate an AWS Provider using roles managed by the infrastructure monorepo. Since Cloud Posse's providers.tf pattern
```

Comment:
```suggestion
# 1. Instantiate an AWS Provider using roles managed by the infrastructure monorepo. This is required because Cloud Posse's `providers.tf` pattern
```